apps/mobile-wallet/src/screens/PublicKeysScreen.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2018 - 2024 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { StackScreenProps } from '@react-navigation/stack'
+import * as Clipboard from 'expo-clipboard'
+
+import { sendAnalytics } from '~/analytics'
+import AddressBadge from '~/components/AddressBadge'
+import { BackButton } from '~/components/buttons/Button'
+import FlatListScreen from '~/components/layout/FlatListScreen'
+import { ScrollScreenProps } from '~/components/layout/ScrollScreen'
+import Row from '~/components/Row'
+import { useAppSelector } from '~/hooks/redux'
+import RootStackParamList from '~/navigation/rootStackRoutes'
+import { selectAllAddresses } from '~/store/addressesSlice'
+import { showToast, ToastDuration } from '~/utils/layout'
+
+interface PublicKeysScreenProps extends StackScreenProps<RootStackParamList, 'PublicKeysScreen'>, ScrollScreenProps {}
+
+const PublicKeysScreen = ({ navigation, ...props }: PublicKeysScreenProps) => {
+  const addresses = useAppSelector(selectAllAddresses)
+
+  const handleAddressPress = async (publicKey: string) => {
+    try {
+      await Clipboard.setStringAsync(publicKey)
+
+      showToast({ text1: 'Public key copied!', visibilityTime: ToastDuration.SHORT })
+      sendAnalytics('Copied public key')
+    } catch (error) {
+      console.log(error)
+      showToast({ text1: 'Error while copying ', visibilityTime: ToastDuration.SHORT, type: 'error' })
+    }
+  }
+
+  return (
+    <FlatListScreen
```

Comment:
Should include `{...props}` in its arguments (see `AddressFlatListScreen` as example)